invenio/legacy/inveniocfg.py

```diff
@@ -832,9 +832,9 @@ def _detect_ip_address():
     """
     try:
         s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        s.connect(('invenio-software.org', 0))
```

Comment:
I understand this is not really part of this patch, but I expect `google.com` to have a greater uptime. Perhaps the issue is mentioning google in the code?